package com.kh.test.exception;

public class Exam29 {

	public static void main(String[] args) {
		System.out.println(new Calculator().divide(12.5, 0));

		System.out.println("프로그램 종료!");
	}

}
